docs/source/examples/zne-braket-ionq.myst

```diff
@@ -0,0 +1,239 @@
+---
+jupytext:
+  text_representation:
+    extension: .myst
+    format_name: myst
+    format_version: 0.13
+    jupytext_version: 1.11.1
+kernelspec:
+  display_name: Python 3
+  language: python
+  name: python3
+---
+
+(label-zne-braket-ionq)=
+# Zero-noise extrapolation with Braket on the IonQ backend
+
+
+This tutorial shows an example of how to apply zero-noise extrapolation ([ZNE](../guide/zne.myst))
+with the [Braket](https://github.com/aws/amazon-braket-sdk-python)
+frontend to mitigate errors on an [IonQ](https://ionq.com/) backend.
+More details on the Mitiq notions of frontends and backends are given [here](../guide/frontends-backends.myst).
+
+Below, we show how to run a simple Braket circuit on an IonQ device, broken down in the following steps.
+
+- [Settings](#settings)
+- [Setup: Defining a circuit](#setup-defining-a-circuit)
+- [High-level usage](#high-level-usage)
+- [Options](#options)
+- [Lower-level usage](#lower-level-usage)
+
++++
+
+## Settings
+
+We import the zero-noise extrapolation module of Mitiq.
+
+```{code-cell} ipython3
+from mitiq import zne
+```
+
+```{code-cell} ipython3
+USE_REAL_HARDWARE = False
+```
+
+**Note:** When `USE_REAL_HARDWARE` is set to `False`, a classically simulated noisy backend is used instead of a real quantum computer.
```

Comment:
```suggestion
```{note}
When `USE_REAL_HARDWARE` is set to `False`, a classically simulated noisy backend is used instead of a real quantum computer.
```
```

Reply:
Suggestion to use the block-style notes as in the "Frontends and Backends" section of the user guide. Unfortunately GitHub won't allow all of the backquote marks to show up in the suggestion.

Reply:
> Unfortunately GitHub won't allow all of the backquote marks to show up in the suggestion.

No problem I manually applied both our suggestions in [8b2cccc](https://github.com/unitaryfund/mitiq/pull/1363/commits/8b2ccccdc73ec434c15cdc7616d9c9a3a380e3c0)